README.md

```diff
@@ -1,53 +1,41 @@
-# AnyCloud
-
-AnyCloud is a Lambda alternative for Node.js that works with multiple cloud providers. We’re building the features of Lambda while fixing the problems we have run into while using it. Namely:
+<div align="center">
+  <img src="./assets/anycloud.png" alt="drawing" width="180"/>
+  <h1>AnyCloud scales webservers in any cloud provider</h2>
+</div>
+<br/>
+
+Our aim is to easily allow developers run their webservers on any cloud provider while also providing a much better experience to alternative services offered in AWS (Lambda, Elastic Beanstalk, ECS/Fargate, EC2) or GCP (Cloud Functions, App Engine, Cloud Run, Compute Engine).
+
+- [x] Automatically scales your HTTP server based on request load and system stats
+- [x] Vendor portability across cloud providers
+- [x] No need to provision and manage virtual machines
+- [x] Runs in your local dev environment as-is
+- [x] HTTPS support included out of the box
+- [x] Supports multi-region and multi-cloud deployments
+- [ ] In-memory distributed datastore
+- [ ] Web socket support
+- [ ] Cron job support
```

Comment:
We'll likely get this before the other two. Should we move it up?

Reply:
I wasn't making much of the order and was waiting for users to start asking. One of the potential users I'm going to demo to on Tuesday (https://www.smarthop.co/) are especially interested because they understood that given our architecture websockets can be supported and for them now having this completely rules out all the Kubernetes / stateless solutions for all of their stack.

Reply:
Websockets working is going to require some work. We probably want to make that a raw TCP socket on the alan/anycloud.ln side and then rely on their language's websocket support, with distributed datastore eliminating the concern of the socket being established with the "wrong" instance.

But that does mean we need to come up with an `@std/tcp` though.

Reply:
I told them it wouldn't be ready a while, but the fact that is was on the roadmap seemed like part of the reason they wanted to look into moving their pure http servers now. That makes sense. I don't know enough of the details for how websockets work, but we should scope it out a little on Monday before I talk to them on Tuesday. Does a websocket opcode akin to the http make sense?